Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -101,26 +95,26 @@ @interface OWSProfileManager ()
 @property (nonatomic, readonly) YapDatabaseConnection *dbConnection;
 @property (nonatomic, readonly) TSNetworkManager *networkManager;
 
+// These properties can be accessed on any thread, while synchronized on self.
```

Comment:
nit: Previously while reading comments making statements about "these properties" I've been confused as to what was actually included in "these properties".  Consider either making a line comment per property or use the specific property names in the comment.